storage/StorageExampleSwiftUI/ImageView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  ImageView.swift
+//  StorageExampleSwiftUI
+//
+//  Created by Ben Reed on 2/10/21.
+//  Copyright © 2021 Google Inc. All rights reserved.
+//
+
+import SwiftUI
+import Firebase
+
+struct ImageView: View {
+    @StateObject private var photoStore = ImageStore(withStorage: Storage.storage())
+    
+    @State var isSelectingImage = false
+    @State var isTakingPhoto = false
+    @State var showUploadMenu = false
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                Image(uiImage: photoStore.image ?? UIImage())
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+            }
+            .navigationTitle("Firebase Storage")
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    if !showUploadMenu {
+                        Button("Upload") {
+                            showUploadMenu = true
+                        }
+                        Spacer()
+                        Button("Download") {
+                            self.downloadImage()
+                        }
+                    } else {
+                        Button("❌") {
+                            showUploadMenu = false
+                        }
```

Comment:
You can use sfsymbols in conjunction with the Button API to make some really clean looking buttons
```swift
  Button(action: {
    showUploadMenu = false
  }, label: {
    Image(systemName: "xmark.circle.fill")
  })
```

Reply:
And side note, maybe remove this button entirely and instead have a "Done" button in the top right of the photo picker. I think  might be a system button that you can enable (i think this functionality provided somewhere for free.. happy to help). It would then look like:
<img width="295" alt="Screen Shot 2021-04-15 at 4 16 13 PM" src="https://user-images.githubusercontent.com/36927374/114932793-e6d9f180-9e05-11eb-9b1f-4fceea0dcf34.png">

Reply:
@ncooke3 Absolutely. This was a temporary fix for the selector. I was hoping to incorporate something a little more exciting and maybe some animation here.